### PR TITLE
Update featured services in API

### PR DESCRIPTION
**Pull request by SanketDG**
Fixes https://github.com/getferdi/server/issues/27

### CODE REVIEW

**Comment by vantezzen** [comment]
Oh, I didn't realize that ferdi-server stores its recipes in the `recipe` folder as well. Could you please move the recipe repository submodule to another folder like `/official_recipes` so we don't clash with user recipes?

**Comment by SanketDG** [comment]
I have updated the code, thanks.

It feels like a hack though, why does ferdi store two folders of recipes?

File: app/Controllers/Http/StaticController.js [hunk 1]
```diff
@@ -99,167 +99,10 @@ class StaticController {
...
-      name: 'twist',
-      version: '1.0.0',
-      icons: {
-        png: 'https://cdn.franzinfra.com/recipes/dist/twist/src/icon.png',
-        svg: 'https://cdn.franzinfra.com/recipes/dist/twist/src/icon.svg',
-      },
-    }]);
+    return response.send(
+      fs
+      .readJsonSync(path.join(Helpers.appRoot(), "recipes", "all.json"))
```
Comment (vantezzen) [inline app/Controllers/Http/StaticController.js:104]: I think this needs to be updated as well?

Comment (SanketDG) [reply app/Controllers/Http/StaticController.js:104]: Done!

**Comment by SanketDG** [comment]
> That is why we have two separate recipe folders: One for the global repository, one for custom recipes.

Makes sense.

There is room for some improvement though, with just one folder for recipes, but separate folders for official and custom (to avoid duplication of data)

**Comment by vantezzen** [comment]
Is this ready to merge?

**Comment by SanketDG** [comment]
> Is this ready to merge?

@vantezzen Yes, this is ready to merge.

**Pull request merged**